frontend/src/ui/player/Paella.tsx

```diff
@@ -119,13 +119,15 @@ const PaellaPlayer: React.FC<PaellaPlayerProps> = ({
             });
 
 
-            const time = new URL(window.location.href).searchParams.get("t");
-            player.bindEvent("paella:playerLoaded", () => {
-                setPlayerIsLoaded(true);
-                if (time) {
-                    player.videoContainer.setCurrentTime(timeStringToSeconds(time));
-                }
-            });
+            if (!isLive) {
+                const time = new URL(window.location.href).searchParams.get("t");
+                player.bindEvent("paella:playerLoaded", () => {
+                    setPlayerIsLoaded(true);
+                    if (time) {
+                        player.videoContainer.setCurrentTime(timeStringToSeconds(time));
+                    }
+                });
+            }
```

Comment:
The `setPlayerIsLoaded(true)` call seems to be unrelated to this right? And thus, it should still happen for live streams? I would probably just add the `!isLive` to the inner `if (time)`. Is a smaller diff as well

Reply:
Yeah technically correct (aka the best kind). I introduced this whole `playerIsLoaded` state solely for setting the timestamp. It has no other purpose, but I guess it might in the future 🤷

Reply:
Ah ok well then we could merge it now.